mjrl_dev/hydra/config/hydra/output/local.yaml

```diff
@@ -1,8 +1,8 @@
 # @package _global_
 hydra:
   run:
-    dir: ./outputs/${hydra.job.name}/${now:%Y-%m-%d_%H-%M-%S}
-    subdir: ${hydra.job.num}_${hydra.job.override_dirname}
+    dir: ./outputs/coss/mjrl/${hydra.job.name}/${now:%Y-%m-%d_%H-%M-%S}
```

Comment:
what's coss?
not everyone will be using mjrl, in which case this can get confusing.